core/templates/pages/topic-editor-page/questions-tab/topic-questions-tab.component.html

```diff
@@ -1,40 +1,41 @@
-<div class="select-skill-dropdown-container" ng-if="!activeQuestion">
+<div class="select-skill-dropdown-container"
+     *ngIf="!activeQuestion">
```

Comment:
I can't find this variable in the component's class. Is it supposed to be `question` maybe?

Reply:
not required since this property wasn't defined previously this condition always resulted in true output